tools/Singularity-devmaster

```diff
@@ -0,0 +1,33 @@
+Bootstrap:docker
+From:debian:latest
```

Comment:
You could just use NeuroDebian: latest and should manual config for datalad

Reply:
Looking at that image I had the impression that it would be much larger. Am I wrong?

I was actually aiming to switch to the slimified debian base image.

Reply:
You are basing on docker. Our docker images are as small as Debian ones, well almost
It is the singularity image is big some should be usable right away